core/server/master/src/main/java/alluxio/master/file/DefaultFileSystemMaster.java

```diff
@@ -1040,6 +1040,10 @@ public void listStatus(AlluxioURI path, ListStatusContext context,
           }
           ensureFullPathAndUpdateCache(inodePath);
 
+          if (context.getOptions().getLoadMetadataOnly()) {
+            continue;
```

Comment:
@apc999 Do you think we should update the auditContext here?
```
auditContext.setSucceeded(true);
```

Similarly, update the metrics too?